Bottom: 79f980111fa2dd27d937a06fd15a2a43db08bcc9
Top:    34484c3f86de0a8b6457f2195b8b38927e2d4963
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-24 03:49:33 +0100

Refresh of add-strip-hierarchy-options-to

---

diff --git a/src/action.c b/src/action.c
index 2ba2d10..5ad05db 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4851,7 +4851,7 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 		EraseElementName ((ElementTypePtr) ptr2);
 		TOGGLE_FLAG (HIDENAMEFLAG, (ElementTypePtr) ptr2);
 /* TODO: 		   SetTextBoundingBox (&PCB->Font, new); */
-		DrawElementName ((ElementTypePtr) ptr2, 0);
+		DrawElementName ((ElementTypePtr) ptr2);
 		Draw ();
 		IncrementUndoSerialNumber ();
 	      }
@@ -4873,7 +4873,7 @@ ActionToggleStripHierarchy (int argc, char **argv, int x, int y)
 		  EraseElementName (element);
 		  TOGGLE_FLAG (STRIPHIERFLAG, element);
 /* TODO: 		   SetTextBoundingBox (&PCB->Font, new); */
-		  DrawElementName (element, 0);
+		  DrawElementName (element);
 		  changed = true;
 		}
 	    }
